Src/FluentAssertions/Specialized/AsyncFunctionAssertions.cs

```diff
@@ -10,11 +10,11 @@ namespace FluentAssertions.Specialized
     /// Contains a number of methods to assert that an asynchronous method yields the expected result.
     /// </summary>
     [DebuggerNonUserCode]
-    public class AsyncFunctionAssertions
+    public abstract class AsyncFunctionAssertions
```

Comment:
Why making this `abstract` without having abstract members?
Please note https://rules.sonarsource.com/csharp/RSPEC-1694

Reply:
You're right. I've reverted my change.